From 46ce2cce2fcab4aa26b020aa2a47d8981ea3f082 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Wed, 16 Jul 2014 16:29:10 +0300
Subject: [PATCH 2/3] gl: Always include EGL_CFLAGS in GL_CFLAGS when EGL_LIBS
 is added to GL_LIBS

They should be handled in tandem, in case any EGL provider could require some
CFLAGS and set them (possibly once moved to prefer pkg-config files),
such as for a custom header location.
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 4b73ebc..8261ee2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1006,6 +1006,7 @@ case $host in
       if test "x$NEED_EGL" != "xno"; then
         if test "x$HAVE_WINDOW_WAYLAND" = "xyes" -o "x$HAVE_WINDOW_X11" = "xyes" -o "x$HAVE_WINDOW_DISPMANX" = "xyes"; then
           GL_LIBS="$GL_LIBS -lEGL $EGL_LIBS"
+          GL_CFLAGS="$GL_CFLAGS $EGL_CFLAGS"
           USE_EGL=yes
         fi
       fi
-- 
1.9.1

